HelpSource/Classes/Server.schelp

```diff
@@ -631,6 +631,35 @@ Show frequency analyzer window.
 method:: meter
 Show input/output level meters.
 
+code::
+// Create a singleton ServerMeter, or bring the window to the front if it has already been created.
+s.meter // -> a ServerMeter
+
+// Query the current position of the ServerMeter window:
+s.meter.position // -> Position(left:5.0, bottom:277.0)
+
+// Query the current bounds of the ServerMeter window:
+s.meter.window.bounds // -> Rect(5.0, 277.0, 134.0, 230.0)
+
+// Move the open ServerMeter to the specified position:
+s.meter.position_(200@200) // -> a ServerMeter
+
+// Retrieve the new position of the ServerMeter window:
+s.meter.position // -> Position(left:200.0, bottom:200.0)
+
+// Retrieve the current bounds of the ServerMeter window:
+s.meter.window.bounds // -> Rect(200.0, 200.0, 134.0, 230.0)
+::
+
+argument:: numIns
+the number of input channels. The default is nil and the number of input channels defined by code::Server.local.options.numInputBusChannels:: is used when using a local server.
+argument:: numOuts
+the number of output channels. The default is nil and the number of output channels defined by code::Server.local.options.numOutputBusChannels:: is used when using a local server.
+argument:: position
+the position of the scope window.
```

Comment:
Note this is the meter window, not the scope (Stethoscope) window.

It would be helpful to explain the point of reference for the position point (bottom left of the meter window I believe?) and the origin of the screen coordinate system.

E.g. setting the position to 150@50 places the bottom left corner of the meter window 150 pixels from the left and 50 pixels up from the bottom of the screen. (I'm assuming...this may not be correct)